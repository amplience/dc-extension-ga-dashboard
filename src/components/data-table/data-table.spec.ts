import { render } from '@testing-library/svelte';
import DataTable from './data-table.svelte';

describe('DataTable', () => {
  it('should render the DataTable Body component', () => {
    const { container } = render(DataTable, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
