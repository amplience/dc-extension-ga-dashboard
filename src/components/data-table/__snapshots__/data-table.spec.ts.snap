// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable should render the DataTable Body component 1`] = `
<div>
  <div
    class="data-table svelte-2e8um5"
  >
    <div
      class="data-table-slot 
    scrollable svelte-2e8um5"
    />
  </div>
</div>
`;
